Dockerfile

```diff
@@ -14,4 +14,11 @@ RUN apt-get update && \
     rm -fr /tmp/sqliteodbc-0.9996 && \
     apt-get purge -y build-essential sqlite3 libsqlite3-dev unixodbc-dev && \
     rm -rf /var/cache/apk/*
-    
+ 
+# Install Python, pip and requests library 
+RUN apt-get update && \
+    apt-get install -y python3-pip python3-dev  && \
+    cd /usr/local/bin && \
+    ln -s /usr/bin/python3 python && \
```

Comment:
I think the installation could be included in the preceding layer without repeatedly running apt-get update.

Reply:
Everything is now included in the same layer